workspace/Dockerfile

```diff
@@ -42,9 +42,9 @@ COPY ./config.template.json /etc/imposm/config.template.json
 COPY osm-initial-import.sh /usr/bin/osm-initial-import
 
 ## Imposm Picker
-COPY ./workspace/imposm-picker/extract-osm-geometry.sh /usr/bin/extract-osm-geometry
```

Comment:
These changes seem correct, OTOH.

Reply:
https://github.com/compose-spec/compose-spec/blob/master/build.md says that when the `build` key is given, it will be the context for all relative paths.  So `./workspace` would become `./workspace/workspace`, I still agree with your change here.

Reply:
(Merged in another patch, https://github.com/wikimedia/kartodock/commit/d491314c24d7f63c1b91896d56d57a4f4785d6be#diff-bc39f31efdfea35cc45b9e33f33ed518029ce035974bc8d6ee1091191040336dR45 )

Reply:
Should we close this PR then?